src/pages/Main/Filter/Filter.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import COLOR_LIST from './colorData';
+import SIZE_LIST from './sizeData';
+import './Filter.scss';
+
+const Filter = () => {
+  const [priceValue, setPriceValue] = useState(50000);
+  const handlePrice = e => {
+    setPriceValue(e.target.value);
+  };
+
+  const [filterData, setFilterData] = useState([]);
+  const filtering = url => {
+    fetch(`http://10.58.52.193:3000/products/all?${url}`, {
+      method: 'GET',
+      headers: { 'Content-Type': 'application/json;charset=utf-8' },
+    })
+      .then(response => response.json())
+      .then(res => setFilterData(res));
+    console.log(filtering);
```

Comment:
```suggestion
    fetch(`http://10.58.52.193:3000/products/all?${url}`)
      .then(response => response.json())
      .then(res => setFilterData(res));
```
fetch 메소드를 사용할 때 GET요청이면 위와 같이 API만 주소만 작성해도 됩니다!
그리고 console.log 제거해주세요!